### PR TITLE
K8syamlrebased

### PR DESCRIPTION
**Pull request by Xaoc000**

### CODE REVIEW

File: content/docs/1.3.0/microservice-catalog/introduction.md [hunk 1]
```diff
@@ -37,18 +37,22 @@ So, to deploy a microservice running those images, we can use the following yaml
...
+kind: Microservice
+metadata:
+  name: Hello Web Microservice
+spec:
+  agent:
+    name: agent-name
+  images:
+    catalogid: 5
+  config: {}
+  roothostaccess: false
```
Comment (Pixcell) [inline content/docs/1.3.0/microservice-catalog/introduction.md]: rootHostAccess

File: content/docs/1.3.0/microservice-catalog/introduction.md [hunk 2]
```diff
@@ -37,18 +37,22 @@ So, to deploy a microservice running those images, we can use the following yaml
...
-application: my-application
+apiVersion: iofog.org/v1
+kind: Microservice
+metadata:
+  name: Hello Web Microservice
+spec:
+  agent:
+    name: agent-name
+  images:
+    catalogid: 5
```
Comment (Pixcell) [inline content/docs/1.3.0/microservice-catalog/introduction.md]: catalogID

File: content/docs/1.3.0/microservice-catalog/using-public-catalog-microservices/jsonrestapi.md [hunk 3]
```diff
@@ -13,23 +13,27 @@ The **config** for the container should look in the following way:
...
+apiVersion: iofog.org/v1
+kind: Microservice
+metadata:
+  name: json-rest-api
+spec:
+  agent:
+    name: local-agent
+    ...
+  images:
+    catalogid: 7
```
Comment (Pixcell) [inline content/docs/1.3.0/microservice-catalog/using-public-catalog-microservices/jsonrestapi.md]: catalogID

File: content/docs/1.3.0/microservice-catalog/using-public-catalog-microservices/jsonrestapi.md [hunk 4]
```diff
@@ -55,19 +59,23 @@ The **config** for the container should look in the following way:
...
+apiVersion: iofog.org/v1
+kind: Microservice
+metadata:
+  name: open-weather-map
+spec:
+  agent:
+    name: local-agent
+    ...
+  images:
+    catalogid: 8
```
Comment (Pixcell) [inline content/docs/1.3.0/microservice-catalog/using-public-catalog-microservices/jsonrestapi.md]: catalogID

File: static/quick-start/linux.sh [hunk 5]
```diff
@@ -239,8 +245,8 @@ do_install_demo() {
 	echo
 	set -x
 
-  $sh_c "curl -L -o dev.tar.gz https://github.com/ioFog/demo/archive/dev-environment.tar.gz" >/dev/null
-  $sh_c "tar -zxvf dev.tar.gz --strip-components=1"
+  sh -c "curl -L -o dev.tar.gz https://github.com/eclipse-iofog/demo/archive/v1.3.0-beta.tar.gz" >/dev/null
```
Comment (Pixcell) [inline static/quick-start/linux.sh]: Before releasing 1.3.0-rc1, we need to update that URL

Comment (Xaoc000) [reply static/quick-start/linux.sh]: Agreed, waiting till after we do develop/doc review this week for stuff like that

**Pull request merged**